action.yml

```diff
@@ -7,6 +7,10 @@ inputs:
   GITHUB_TOKEN:
     description: 'A GitHub token.'
     required: true
+  BOT_NAME:
+    description: 'The bot name on which you want the auto-merge to be enabled'
```

Comment:
There is no real limitation to not use a `bot` account. So this is the Github login that will have the PRs auto merged